src/alloc/isolated_alloc.rs

```diff
@@ -183,6 +203,8 @@ impl IsolatedAlloc {
         let (size, align) = IsolatedAlloc::normalized_layout(layout);
 
         if IsolatedAlloc::is_huge_alloc(size, align, self.page_size) {
+            // SAFETY: Partly upheld by caller, and we checked that the size
+            // and align mean this must have been allocated via `alloc_huge`
```

Comment:
"mean" -> "meaning"?

Reply:
It was intentional but I realise it sounds kinda weird, I'll edit it